EIPS/eip-7092.md

```diff
@@ -300,7 +300,7 @@ the `balance` of an account MUST be equal to the ratio of the `principal` of tha
     balanceOf(account) = principalOf(account) / denomination
 ```
 
-### Backwards compatibility
+## Backwards compatibility
```

Comment:
must be `## Backwards Compatibility`

Reply:
Hello @lightclient 

When I use `## Backwards Compatibility` the test fails. The error message is 'An extra section has been added'. The backward compatibility doesn't appear in the template, that's probably the reason why when we add it as a section with ## the test fails

Reply:
Yes, I think there was a typo in the previous one. the previous one was `Backwards compatibility`, but it should be `Backwards Compatibility` with capital `C`.

I just corrected it.